pkgs/applications/science/biology/sortmerna/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "sortmerna";
+  version = "2.1";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "biocore";
+    rev = "${version}";
+    sha256 = "0ghqjn70sahan7g1adl01r9gkk59q8jams837pbkcb4x7jf8yqkk";
+  };
+
+  buildInputs = [ zlib ];
+
+  # The `make install` command shipped with the package installs header files,
+  # which are not necessary:
+  installPhase = ''
```

Comment:
I think it would be cleaner to just use the default `installPhase` and just remove the headers in `postFixup`?